mobile/lib/features/trade/async_order_change_notifier.dart

```diff
@@ -27,8 +27,11 @@ class AsyncOrderChangeNotifier extends ChangeNotifier implements Subscriber {
 
   @override
   void notify(bridge.Event event) {
-    if (event is bridge.Event_BackgroundNotification &&
-        event.field0 is bridge.BackgroundTask_AsyncTrade) {
+    if (event is bridge.Event_BackgroundNotification) {
+      if (event.field0 is! bridge.BackgroundTask_AsyncTrade) {
+        // ignoring other kinds of background tasks
+        return;
+      }
```

Comment:
The background notification can be anything. In this scenario we are only interested in background notifications to the async trade, but I can only subscribe to the event itself. In order to prevent printing confusing "received unexpected event" messages, I have changed the condition to this.

It wasn't a problem with combining `&&` and `is`;)